src/dns.ml

```diff
@@ -2203,7 +2368,7 @@ module Rr_map = struct
   let to_int : type a. a key -> int = function
     | A -> 1 | Ns -> 2 | Cname -> 5 | Soa -> 6 | Ptr -> 12 | Mx -> 15
     | Txt -> 16 | Aaaa -> 28 | Srv -> 33 | Ds -> 43 | Sshfp -> 44
-    | Rrsig -> 46 | Nsec -> 47 | Dnskey -> 48 | Nsec3 -> 50
+    | Rrsig -> 46 | Nsec -> 47 | Dnskey -> 48 | Nsec3 -> 50 | Loc -> 29
```

Comment:
could we move that to a place so that the RHS are ordered by integers? i.e. a line above in the middle